### DISCUSSION TITLE
Compiling with respect to root directory

### DISCUSSION THREAD

**Discussion by jonhue** [James-Yu/LaTeX-Workshop#3352] [Q&A] [answered]
Hey there, thanks for this great extension!

I have a project which consists of multiple "root" TeX files nested in folders,

```
folder1
|-- file1.tex
folder2
|-- file2.tex
folder3
|-- file3.tex
settings.tex
```

However, I would like all file paths to be relative to the root directory. That is, when including `settings.tex` from `file1.tex`, I want to write `\input{settings}` rather than `\input{../settings}`.

As far as I understand, this requires that the compilation is run from the root folder. Is this possible?

The reason for why I want to do it this way is that I have an environment downstream, which is setup this way.

**Comment by James-Yu** [comment, marked as answer]
This seems not an extension related problem. May look into the `imports` package? I'm not familiar with that.
